pyproject.toml

```diff
@@ -13,6 +13,7 @@ dependencies = [
     "click >= 8.0.1, < 9",
     "colorama ~= 0.4.6",
     "jinja2 ~= 3.1.2",
+    "markdown-it-py",
```

Comment:
It's okay to not specify a lower bound for the version, but we should specify a `< 5` for the upper bound